common/src/main/java/org/figuramc/figura/lua/docs/FiguraGlobalsDocs.java

```diff
@@ -110,6 +110,35 @@ public abstract class FiguraGlobalsDocs {
     )
     public static Object require() {return null;}
 
+    @LuaMethodDoc(
+            overloads = {
+                    @LuaMethodOverload(
+                            argumentTypes = String.class,
+                            argumentNames = "scriptName"
+                    ),
+                    @LuaMethodOverload(
+                            argumentTypes = {String.class, String.class},
+                            argumentNames = {"scriptName", "scriptContents"}
+                    )
+            },
+            value = "globals.add_script"
+    )
+    public static Object addScript() {return null;}
+
+    @LuaMethodDoc(
+            value = "globals.get_scripts"
+    )
+    public static Object getScripts() {return null;}
+
+    @LuaMethodDoc(
+            overloads = @LuaMethodOverload(
+                    argumentTypes = String.class,
+                    argumentNames = "scriptName"
+            ),
+            value = "globals.get_script"
+    )
+    public static Object getScript() {return null;}
```

Comment:
Except for one thing: 
I know the other methods are also formatted like this but it just feels... wrong.

```suggestion
    public static Object getScript() {
        return null;
    }
```